src/com/esotericsoftware/kryo/Kryo.java

```diff
@@ -225,6 +227,12 @@ public Kryo (ClassResolver classResolver, ReferenceResolver referenceResolver) {
 		OptionalSerializers.addDefaultSerializers(this);
 		TimeSerializers.addDefaultSerializers(this);
 		ImmutableCollectionsSerializers.addDefaultSerializers(this);
+		// Add RecordSerializer if JDK 14+ available
+		if (isClassAvailable("java.lang.Record")) {
+			try {
+				addDefaultSerializer(Class.forName("java.lang.Record"), RecordSerializer.class);
+			} catch (ClassNotFoundException ignored) {} // handled in if-clause
+		}

```

Comment:
Could you please add a helper method for this, so the registration is easier on the eyes?

    public void addDefaultSerializer (String className, Class<? extends Serializer> serializerClass) { ... }